ascon/Cargo.toml

```diff
@@ -1,34 +1,37 @@
 [package]
 name = "ascon"
-version = "0.2.0"
-description = """
-Pure Rust implementation of Ascon, a family of authenticated encryption and
-hashing algorithms designed to be lightweight and easy to implement
-"""
-authors = ["quininer kel <quininer@live.com>", "RustCrypto Developers"]
+version = "0.3.0-pre"
+description = "Pure rust implementation of the Ascon permutation"
+authors = [
+    "Sebastian Ramacher <sebastian.ramacher@ait.ac.at>",
+    "RustCrypto Developers",
+]
 license = "Apache-2.0 OR MIT"
 documentation = "https://docs.rs/ascon"
 repository = "https://github.com/RustCrypto/sponges/tree/master/ascon"
-keywords = ["aead", "crypto", "sponge"]
+keywords = ["Ascon", "crypto", "permutation"]
 categories = ["cryptography", "no-std"]
 readme = "README.md"
 edition = "2021"
 rust-version = "1.60"
 
-[dependencies]
-aead = { version = "0.5", optional = true, default-features = false }
-subtle = { version = "2", optional = true, default-features = false }
-
 [dev-dependencies]
-hex-literal = "0.3"
+criterion = "0.4"
```

Comment:
Criterion is a relatively heavy dependency, so we usually put criterion benchmarks into a separate crate outside of usual CI runs. For example, see the structure of the [stream ciphers](https://github.com/RustCrypto/stream-ciphers) repository.

Reply:
Done

Reply:
criterion takes <30s to build on my laptop, FWIW, and the resulting artifacts are cached so that's effectively a one-time (or failing that, infrequent) cost.

While at one point might've been worth it, I'm not sure it is anymore.